vividus-extension-selenium/src/main/java/org/vividus/selenium/cloud/AbstractCloudTestStatusManager.java

```diff
@@ -78,6 +88,24 @@ public final void updateCloudTestStatus(BeforeWebDriverQuitEvent event)
         }
     }
 
+    @Subscribe
+    public final void waitForSessionCompletion(AfterWebDriverQuitEvent event)
+    {
+        String sessionId = event.sessionId();
+        boolean completed = sessionCompletionWaiter.wait(() -> isTestExecutionComplete(sessionId), r -> r);
+        if (!completed)
+        {
+            LOGGER.atError().addArgument(SESSION_COMPLETE_TIMEOUT_SECONDS)
+                    .addArgument(sessionId)
+                    .log("Timeout while waiting for cloud test session completion after {} seconds. SessionId: {}");
```

Comment:
```suggestion
                    .log("Timeout while waiting for SauceLabs test job completion after {} seconds. SessionId: {}");
```

Reply:
I prefer to leave this message as is because it can be reused for example for browserstack or other cloud test service